src/ucp/core/ucp_context.c

```diff
@@ -253,6 +253,13 @@ static ucs_config_field_t ucp_config_table[] = {
    "of all entities which connect to each other are the same.",
    ucs_offsetof(ucp_config_t, ctx.unified_mode), UCS_CONFIG_TYPE_BOOL},
 
+  {"SOCKADDR_CM_WIREUP", "n",
```

Comment:
maybe it should be ternary value
yes = require a CM-capable component
try = use a CM-capable component if exists
no = disable

rename to "SOCKADDR_CM_ENABLE"